<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flags</class>
 <widget class="QDialog" name="Flags">
  <property name="windowTitle">
   <string>Drapeaux du paquet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Les drapeaux permettent de spécifier à Logram comment gérer certains paquets. Ils vous permettent par exemple de ne jamais installer un paquet dont vous savez qu'il pose problème.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Drapeaux</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblDesktopIntegration">
          <property name="text">
           <string>Intégration à l'environnement :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblKDEIntegration"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chkGUI">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Application graphique</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkPrimary">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Paquet principal construit par sa source</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDontInstall">
        <property name="text">
         <string>Ne pas installer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDontRemove">
        <property name="text">
         <string>Ne pas supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDontUpdate">
        <property name="text">
         <string>Ne pas mettre à jour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkEULA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Nécessite l'approbation d'une licence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkReboot">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Nécessite un redémarrage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkInstalled">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Installé</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRemoved">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Supprimé</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkWanted">
        <property name="text">
         <string>Conserver même si plus nécessaire</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Flags</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Flags</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
